src/plugins/mer/meremulatormodedialog.cpp

```diff
@@ -222,7 +225,8 @@ bool MerEmulatorModeDialog::execDialog()
         m_ui->restartEmulatorCheckBox->setEnabled(false);
     }
 
-    if (m_dialog->exec() != QDialog::Accepted) {
+    int dialogResult = m_dialog->exec();
```

Comment:
Since we do not use it for anything else later, I would consider just moving the original line

`const bool result = m_dialog->result() == QDialog::Accepted;`

here.